faq/font_sample.php

```diff
@@ -85,16 +55,16 @@
     show_font_specimen($proofreading_font, $proofreading_font);
 }
 
-echo "<h2 id='DPCustomMono2' style='clear: both;'>DPCustomMono2</h2>";
+echo "<h2 id='DPSansMono' style='clear: both;'>DP Sans Mono</h2>";
 
-echo "<p style='clear: both;'>" . _("DPCustomMono2 is a font adapted by DP volunteers, based on the suggestions and ideas of many experienced proofreaders. Using DPCustomMono2 as your proofreading font can help you find mistakes."). "</p>\n";
+echo "<p style='clear: both;'>" . _("DP Sans Mono is a font adapted by DP volunteers, based on the suggestions and ideas of many experienced proofreaders. Using DP Sans Mono as your proofreading font can help you find mistakes."). "</p>\n";
 
-echo "<p style='clear: both'>" . _("This site provides DPCustomMono2 as a web font and browsers that support web fonts do not require the font to be installed locally.") . " ";
-echo sprintf(_("For more information on installing and using the font locally for offline activitlies like Post-Processing and Smooth Reading, see the <a href='%s'>DPCustomMono2</a> wiki page at pgdp.net."), "http://www.pgdp.net/wiki/Installing_DPCustomMono") . "</p>\n";
+echo "<p style='clear: both'>" . _("This site provides DP Sans Mono as a web font and browsers that support web fonts do not require the font to be installed locally.") . " ";
+echo sprintf(_("For more information on installing and using the font locally for offline activitlies like Post-Processing and Smooth Reading, see the <a href='%s'>DP Sans Mono</a> wiki page at pgdp.net."), "https://www.pgdp.net/wiki/DP_Sans_Mono") . "</p>\n";
```

Comment:
The "[DPCustomMono2 Font](https://www.pgdp.net/wiki/DP_Official_Documentation:Proofreading/DPCustomMono2_Font)" wiki page is in the DP Official Documentation namespace; probably the `DP Sans Mono` page ought to be, too? (Or at least protected.)

Reply:
I want a short URL so we'll go with this one (See also: https://www.pgdp.net/wiki/DPCustomMono2&redirect=no). I haven't decided if I want to make protected or a redirect to an DPOD yet but we can do one or the other.